make-release.sh

```diff
@@ -112,11 +112,11 @@ set -e
 sed -i version/version.go -r -e "s#(Version = \")(v?[0-9.]+)(\")#\1${VERSION}\3#g"
 # change image tag in Makefile
 DWO_QUAY_IMG="quay.io/devfile/devworkspace-controller:${VERSION}"
-sed -i Makefile -r -e "s#quay.io/devfile/devworkspace-controller:.*#${DWO_QUAY_IMG}#g"
+sed -i Makefile -r -e "s#quay.io/devfile/devworkspace-controller:.*[0-9a-zA-Z._-]#${DWO_QUAY_IMG}#g"
```

Comment:
I'm not sure I understand this regex, is there an example of what kinds of replacement are failing? This will still do something unexpected if e.g. the line is
```
ifeq ($(DWO_IMG),quay.io/devfile/devworkspace-controller:next) # Check if image...
```

We could instead just match the valid chars for an image tag:
```suggestion
sed -i Makefile -r -e "s#quay.io/devfile/devworkspace-controller:[0-9a-zA-Z._-]+#${DWO_QUAY_IMG}#g"
```

Reply:
you're right, your version is more correct, thanks